doc/classes/AnimatedSprite2D.xml

```diff
@@ -17,7 +17,7 @@
 			<param index="0" name="anim" type="StringName" default="&amp;&quot;&quot;" />
 			<param index="1" name="backwards" type="bool" default="false" />
 			<description>
-				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [code]backwards[/code] is [code]true[/code], the animation will be played in reverse.
+				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [param backwards] is [code]true[/code], [member speed_scale] is set to a negative value, and the animation is played in reverse.
```

Comment:
>If [param backwards] is [code]true[/code], [member speed_scale] is set to a negative value

Technically that's not true if `speed_scale` is zero (unless someone interpretes "negative" as `<= 0` (e.g. Akien does so :stuck_out_tongue_winking_eye:), then it would be true). But yeah, it's probably fine as is and mentioning zero `speed_scale` case would just make it more convoluted (but I had to mention it in here :slightly_smiling_face:).

Reply:
There could be a spot to mention it in `speed_scale`, actually. Setting `speed scale` to 0 is not the same as stopping the animation or setting the frame manually. It keeps the internal timeout intact until `speed_scale` is changed again, which is pretty beneficial in some situations (think of a dramatic freeze frame) for consistent results.

Reply:
Actually I do recall some user(s?) asking for how to pause/resume an AnimaterSprite properly (respecting the already elapsed time for the current frame), I'm not sure if I thought about using `speed_scale` for it back then. `play`/`stop` of course don't fulfill such use case. So it indeed might be worth mentioning somewhere (e.g. in `speed_scale`'s description and `stop`'s description could refer to it for how to pause like that (as user would most likely check out `stop` when looking for pausing)).

Reply:
Honestly, and I am repeating myself, AnimatedSprite is functional, but feels old. I'm not going to change the behaviour in this PR, for example, but I do feel like setting `playing` to **false** should **not**  be the equivalent of `stop()` (it shouldn't reset the internal frame time out). There's other minor quirks like these in the class, probably worth addressing and I can't tell if they're all bugs.

I will update the description to mention the 0 behaviour as best as I can, but it could prove difficult to explain a "mid-frame time-out".